cogs/induct.py

```diff
@@ -479,7 +474,7 @@ async def ensure_members_inducted(self, ctx: TeXBotApplicationContext) -> None:
                 await member.add_roles(
                     guest_role,
                     reason=(
-                        f"{ctx.user} used TeX Bot slash-command: \"/ensure-members-inducted\""
+                        f'{ctx.user} used TeX Bot slash-command: "/ensure-members-inducted"'
```

Comment:
Prefer strings to use double quotes & escape interior double quotes rather than using single quotes